cmd/importer/rand.go

```diff
@@ -80,7 +81,7 @@ func randDate(col *column) string {
 
 	minTime, err := time.Parse(dateFormat, min)
 	if err != nil {
-		log.Warnf("randDate err %s", err)
+		log.Warn("randDate", zap.Error(err))
```

Comment:
Do we need these warning msgs? @lamxTyler

Reply:
Cureently yes. Maybe it is better to return the error to caller, but it is another issue.

Reply:
any better idea for the error message?

Reply:
"parse min date failed"?

Reply:
Done.
PTAL @lamxTyler @XuHuaiyu